mockfirestore/main.py

```diff
@@ -144,6 +144,11 @@ def stream(self) -> Iterator[DocumentSnapshot]:
                 doc_snapshots = sorted(doc_snapshots,
                                        key=lambda doc: doc.to_dict()[key],
                                        reverse=direction == 'DESCENDING')
+            if self._start_at:
+                doc_snapshots = self._cursor_helper(self._start_at[0], doc_snapshots, self._start_at[1], True)
+
+            if self._end_at:
+                doc_snapshots = self._cursor_helper(self._end_at[0], doc_snapshots, self._end_at[1], False)
```

Comment:
```suggestion
                document_fields, before = self._start_at
                doc_snapshots = self._cursor_helper(document_fields, doc_snapshots, before, False)
```

Reply:
changed to `document_fields, before = self._end_at` following the convention above.  renamed `after` to `before` for consistency since `_apply_cursor(..)` takes `before` as an arg.